### DISCUSSION TITLE
Tab cloak

### DISCUSSION THREAD

**Discussion by Terrinn** [3kh0/ext-remover#264] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How can I make a tab cloak for my games?

**Comment by 3kh0** [comment, marked as answer]
Ask that here: https://discord.gg/wv6huJAwEv

**Reply by Terrinn** [reply]
Invalid link

**Reply by 3kh0** [reply]
@Terrinn do not use a proxy

**Reply by Terrinn** [reply]
What should I do then? @3kh0

**Reply by FilianFromPhotop** [reply]
heres a tab cloaker: javascript:function gcloak() { var link = document.querySelector("link[rel*='icon']") || document.createElement('link');link.type = 'image/x-icon';link.rel = 'shortcut icon';link.href = 'https://ssl.gstatic.com/docs/doclist/images/infinite_arrow_favicon_5.ico';document.title = 'My Drive - Google Drive';console.log(document.title);document.getElementsByTagName('head')[0].appendChild(link) };gcloak();setInterval(gcloak, 1000);

**Reply by rwg100** [reply]
the code doesnt work
